.golangci.yaml

```diff
@@ -1,17 +1,20 @@
 # golangci-lint configuration file
 # see: https://golangci-lint.run/usage/configuration/
 
-# Options for analysis running
-run:
-  # Which dirs to skip: they won't be analyzed;
```

Comment:
`run.skip-dirs` has been deprecated since golangci-lint v1.57 and replaced by `issues.exclude-dirs`. These options are technically identical. For a detailed explanation, see https://github.com/golangci/golangci-lint/pull/4509.

`golangci-lint` does not provide a way to skip directory analysis, `run-skip-dirs` lies.